tdcommon/src/main/java/thredds/server/catalog/ConfigCatalogCache.java

```diff
@@ -107,7 +111,7 @@ public ConfigCatalog get(final String catKey) throws IOException {
     }
   }
 
-  static public ConfigCatalog readCatalog(String catalogFullPath) throws IOException {
```

Comment:
Is it okay to make this not static since it is a public function and I guess could be used outside of TDS?

Reply:
does it need to be not static?

Reply:
oooh, right

Reply:
either way changes the signature though

Reply:
as discussed, left static and made version with default context to avoid changing signature in public api (3804c6689f)